stan/math/prim/scal/prob/cauchy_lcdf.hpp

```diff
@@ -20,6 +20,23 @@
 namespace stan {
   namespace math {
 
+    /**
+     * Calculates the cauchy log cumulative distribution function for
+     * the given variate, location, and scale.
+     *
+     * \f$\frac{1}{\pi}\arctan\left(\frac{y-\mu}{\sigma}\right) + \frac{1}{2}\f$
```

Comment:
I'm opposed to the formulae unless we do something really interesting, I pulled this in because the cauchy was already partially documented and had these formulas in. will cut it here and where I see it.